app/Components/App.js

```diff
@@ -35,15 +35,14 @@ class App extends React.Component {
     }
   }
 
-  createGame(e) {
-    e.preventDefault();
+  createGame() {
     socket.emit("newgame");
   }
 
   render() {
     return (
       <div>
-        <button onClick={this.createGame}> Create New Game </button>
+        <button onClick={() => this.createGame()}> Create New Game as Student </button>
```

Comment:
This makes a new function that then calls `createGame`.  This doesn't really cause any harm, but you can just keep it as `onClick={this.createGame}` (or `this.onCreateGameClicked`).